README.Rmd

```diff
@@ -69,6 +69,14 @@ rmarkdown::draft(
 rmarkdown::render("my_preregistration.Rmd")
 ```
 
+## Uploading your preregistration
+After knitting your preregistration to a PDF file using this package, you need to upload this protocol to a trustworthy repository to complete your preregistration.
+Possible repositories for this are:
+
+- Preregistration in Psychology (https://prereg-psych.org/): A preregistration platform provided by the Leibniz Institute for Psychology (ZPID) which focuses on psychological research
```

Comment:
Nice. Minor quibble: Could you turn the names into links (e.g., `[Preregistration in Psychology](https://prereg-psych.org/)`)?